code/modules/mob/living/simple_animal/abnormality/_tools/zayin/wishwell.dm

```diff
@@ -283,3 +286,54 @@
 		playsound(src, 'sound/abnormalities/bloodbath/Bloodbath_EyeOn.ogg', 80, TRUE, -3)
 		new gift(get_turf(src))
 		visible_message("<span class='notice'>Something comes out of the well!</span>")
+
+//Throw yourself into the well : The Code
+/obj/structure/toolabnormality/wishwell/user_buckle_mob(mob/living/M, mob/user, check_loc = TRUE)
+	if(M != user)
+		return FALSE
+
+	if (!istype(M, /mob/living/carbon/human))
+		to_chat(usr, "<span class='warning'>It doesn't look like I can't quite fit in.</span>")
+		return FALSE // Can only extract from humans.
+
+	to_chat(user, "<span class='warning'>You start climbing into the well.</span>")
+	if(!do_after(user, 7 SECONDS))
+		to_chat(user, "<span class='notice'>You decide that might be a bad idea.</span>")
+		return FALSE
+
+	to_chat(user, "<span class='userdanger'>You fall into the well!</span>")
+	return ..(M, user, check_loc = FALSE) //it just works
+
+/obj/structure/toolabnormality/wishwell/post_buckle_mob(mob/living/carbon/human/M)
+	if(!ishuman(M))
+		return
+	var/deathgift
+	var/stat_total
+	for(var/attribute in M.attributes)
+		stat_total += get_raw_level(M, attribute)
+
+	switch(stat_total)
```

Comment:
I suggest rounding this number, stats display as whole numbers but code-wise tend to have a lot of decimals unless at the cap.